tasks/natural-language-processing/sentiment-analysis/Pritam/summary_file.txt

```diff
@@ -0,0 +1 @@
+The net effect is re-allocation of computational resources from less efficient parts of the network to parts of the network where they might be more useful. The shrinking phase of MorphNet is useful to produce a neural network that optimizes the cost for a specific resource. That�s precisesly why MorphNet uses an expanding phase based on a width multiplier to expand the sizes of all layers.
```

Comment:
File has not UTF-8 characters causing error running mlflow with default parameters. Could you please check. I believe quotation in "That’s" is causing problem.